hops/exceptions.py

```diff
@@ -9,4 +9,7 @@ class UnkownSecretStorageError(Exception):
     """This exception will be raised if an unused secrets storage is passed as a parameter"""
 
 class APIKeyFileNotFound(Exception):
-    """This exception will be raised if the file with the API Key is not found in the local filesystem"""
\ No newline at end of file
+    """This exception will be raised if the file with the API Key is not found in the local filesystem"""
+
+class ServingAPIKeyNoFound(Exception):
```

Comment:
ServingAPIKeyNoFound should be ServingAPIKeyNotFound

Reply:
done